core/document.adoc

```diff
@@ -0,0 +1,70 @@
+= OGC API - Processes - Part 1: Core
+:doctype: standard
+:docsubtype: Interface
+:edition: 1.0
+:language: en
+:docnumber: 18-062
+:status: draft
+:copyright-year: 2018
+:received-date: yyyy-mm-dd
+:issued-date: yyyy-mm-dd
+:published-date: yyyy-mm-dd
+:fullname: Benjamin Pross
+:fullname_2: Panagiotis (Peter) A. Vretanos
+:keywords: ogcdoc, OGC document, OGC API, Geospatial API, processes, Web Processing Service, WPS, JSON, HTML, geoprocessing, API, OpenAPI, HTML
+:submitting-organizations: 52°North GmbH; Hexagon; CubeWerx Inc.; Ecere Corporation; Terradue Srl; European Space Agency (ESA); Spacebel
+:local-cache-only:
+:data-uri-image:
+:docfile: document.adoc
```

Comment:
Remove, this is useless.

Reply:
@ronaldtse, which attributes are you referring to?
`:submitting-organizations:`? `:local-cache-only:`? `:data-uri-image:`? `:docfile:`?
Or all four?

Reply:
`:docfile:`. We used to use this for the site command but it is no longer used.

Reply:
OK, done. Thanks!

Reply:
Thanks!